mars-sim-core/src/main/java/com/mars_sim/core/equipment/EquipmentFactory.java

```diff
@@ -142,39 +109,39 @@ public static double getEquipmentMass(EquipmentType type) {
 	 * @param processName
 	 * @return
 	 */
-    public static double calculateMass(String processName, String productName) {	
-		if (weights.isEmpty() || !weights.containsKey(processName)) {
-			double mass = 0;
-			double quantity = 1;
-	    	ManufactureProcessInfo manufactureProcessInfo = null;
-
-	    	if (manufactureConfig == null) {
-	    		manufactureConfig = SimulationConfig.instance().getManufactureConfiguration();
-	    	}
-	    	
-	    	for (ManufactureProcessInfo info : manufactureConfig.getManufactureProcessList()) {
-	    		if (info.getName().equalsIgnoreCase(processName)) {
-	    			manufactureProcessInfo = info;
-	    			break;
-		        }
-	    	}
+    private static double calculateMass(EquipmentType type) {
+		
+		// Note: it's haphazard to match the string of the manu process since it can change.
+		// Will need to implement a better way in matching and looking for the manu process that assemblies the item of interest.
+		String processName = switch(type) {

```

Comment:
So we may as well find a way to look for and flag these manufacturing processes right away when processing ManufactureConfig at the start of the sim. 

They will be `ManufactureProcessInfo` instances.

If we can create a class for flagging these processes, we can use it for those haphazard process name strings in ItemResourceUtil as well.

Reply:
Possible yes. Key point it works and the new unit test will tell us when it stops working.
It's part of the bigger question of how to handle locale in the configurations, but that's not that important